classes/Domain/Model/TalkMeta.php

```diff
@@ -8,6 +8,12 @@ class TalkMeta extends Model
 {
     protected $table = 'talk_meta';
     public $timestamps = false;
+    /**
+     * The attributes that aren't mass assignable.
+     *
+     * @var array
+     */
+    protected $guarded = [];
```

Comment:
Anytime I use Eloquent, the first thing I do is make a base model that does exactly this. We could go ahead and do that: `OpenCFP\Domain\Model\Model`.